docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,392 @@
+             --project PROJECT_NAME               \
+             --service-type flink                 \
+             -s demo-postgresql                   \
+             -d demo-flink
+
+   c. Enter the following command to list the integrations:
+
+      ::
+
+         avn service integration-list demo-flink
+
+      The output should show you that both ``demo-kafka`` and ``demo-postgresql`` integrations are enabled as well as the corresponding ``integration_id`` values that you need later when creating data tables.
+
+
+
+Set up sample data
+------------------
+
+Before you start, clone the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ Git repository to your computer and then run ``pip install -r requirements.txt`` in your local copy to make sure that the necessary dependencies are installed.
+
+1. Using the Aiven CLI, run the following command to download the certificate files for your ``demo-kafka`` service:
+
+   ::
+
+      avn service user-creds-download demo-kafka  \
+          --username avnadmin                     \
+          -d TARGET_DIR
+
+   Replace ``TARGET_DIR`` with the path to a local folder on your computer.
+
+#. Run the following command to see the service parameters, then copy the ``host`` and ``port`` values.
+
+   ::
+
+      avn service get kafka-demo --format '{service_uri_params}'
+
+#. Run the following Python command to create the sample records using the `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_ tool:
+
+   ::
+
+      python3 python-fake-data-producer-for-apache-kafka/metricproducer.py \
+          --cert-folder DOWNLOADED_CERTIFICATE_FOLDER \
+          --host KAFKA_HOST_ADDRESS \
+          --port KAFKA_PORT \
+          --topic-name cpu_load_stats_real \
+          --nr-messages 0 \
+          --max-waiting-time 1
+
+
+   Replace ``DOWNLOADED_CERTIFICATE_FOLDER`` with the folder that contains the three certificate files that you downloaded, and ``KAFKA_HOST_ADDRESS`` and ``KAFKA_PORT`` with the address and port for your Aiven for Apache Kafka service.
+
+   .. note::
+      The ``--nr-messages 0`` option creates a continuous flow of messages that never stops.
+
+   This command pushes the following type of events to the ``cpu_load_stats_real`` topic in your Kafka service:
+
+   ::
+   
+      {"hostname": "dopey", "cpu": "cpu4", "usage": 98.3335306302198, "occurred_at": 1633956789277}
+      {"hostname": "sleepy", "cpu": "cpu2", "usage": 87.28240549074823, "occurred_at": 1633956783483}
+      {"hostname": "sleepy", "cpu": "cpu1", "usage": 85.3384018012967, "occurred_at": 1633956788484}
+      {"hostname": "sneezy", "cpu": "cpu1", "usage": 89.11518629380006, "occurred_at": 1633956781891}
+      {"hostname": "sneezy", "cpu": "cpu2", "usage": 89.69951046388306, "occurred_at": 1633956788294}
+
+
+Create a pipeline for basic filtering
+-------------------------------------
+
```

Comment:
Can we add a quick diagram stating what we do with SOURCE->FLINK_JOB_NAME->TARGET?

Reply:
Yep, I've added those to both console and CLI versions of the instructions ;)